SpacetimeDB/crates/bench/hyper_cmp.py

```
@@ -0,0 +1,174 @@
+# Mini-tool for comparing benchmark between PR or locally
+import os
+import json
+import argparse
+import subprocess
+import shutil
+
+
+def statDecoder(statDict):
+    return namedtuple('Stat', statDict.keys())(*statDict.values())
```

Pylint (3.3.7)

[error] Undefined variable 'namedtuple'

(E0602)